server/src/main/java/org/apache/cassandra/sidecar/routes/restore/RestoreJobSummaryHandler.java

```diff
@@ -50,6 +59,17 @@ public RestoreJobSummaryHandler(ExecutorPools executorPools,
         super(instanceMetadataFetcher, executorPools, validator);
     }
 
+    @Override
+    public Set<Authorization> requiredAuthorizations()
+    {
+        String resource = VariableAwareResource.DATA_WITH_KEYSPACE_TABLE.resource();
+        Authorization createRestore = SidecarPermissions.CREATE_RESTORE_JOB.toAuthorization(resource);
+        Authorization viewRestore = SidecarPermissions.VIEW_RESTORE_JOB.toAuthorization(resource);
+        OrAuthorization createOrView
+        = new OrAuthorizationImpl().addAuthorization(createRestore).addAuthorization(viewRestore);
+        return ImmutableSet.of(createOrView);
```

Comment:
Same, why allow `create`?

Reply:
I thought if someone has create permission for restore job, then they should be able to view summary. I can remove create from update handler but for view it makes sense?

Reply:
Discussed this offline, decided to have just view permission here.